client/src/Pages/Dashboard/DashboardView.js

```diff
@@ -17,15 +16,18 @@ import TravelDistanceTotalCard from "../../Components/Card/TravelDistanceTotalCa
 import TimeInCarCard from "../../Components/Card/TimeInCarCard";
 import EmissionsCard from "../../Components/Card/EmissionsCard";
 import FuelEconomyCard from "../../Components/Card/FuelEconomyCard";
-
 import GoogleMap from "../../Components/GoogleMap/GoogleMap";
 import AntdTableComponent from "../../Components/AntdTable/AntdTableComponent";
 import Footer from "../../Components/Footer";
-
 export default function DashboardView(props) {
-  const { loading, error, data } = useQuery(getDashboardAllDataQuery, {
+    const [selectedOption , setSelectedOption] = useState(null)
```

Comment:
the variable names are a bit generic right now. Suggest making it clearer what the variable is for.